From 705739fa54260b7a0e6cbba0b5a99e52c79f9c09 Mon Sep 17 00:00:00 2001
From: David Allsopp <david.allsopp@metastack.com>
Date: Tue, 6 Aug 2019 09:23:06 +0100
Subject: [PATCH] Fix failure to install tools links

In --disable-installing-bytecode-programs mode, the .opt version of the
tools is installed, but the symlink for the tool itself is not created.
---
 tools/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/Makefile b/tools/Makefile
index 530dd37f34..1b3014a3ab 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -197,6 +197,7 @@ else
 	do \
 	  if test -f "$$i".opt; then \
 	    $(INSTALL_PROG) "$$i.opt" "$(INSTALL_BINDIR)/$$i.opt$(EXE)"; \
+	    (cd "$(INSTALL_BINDIR)/" && $(LN) "$$i.opt$(EXE)" "$$i$(EXE)"); \
 	  fi; \
 	done
 endif
-- 
2.20.1

